pubspec.yaml

```diff
@@ -120,6 +120,11 @@ dependency_overrides:
       url: https://github.com/MixinNetwork/bitsdojo_window.git
       ref: 78faaa6440c7fee146258bb0ef3336c33ed4555d
       path: bitsdojo_window_windows
+  drift:
```

Comment:
https://pub.dev/packages/drift/versions  drift just has release 1.5  17 hours ago, so can we use the release version directly?

Reply:
Let me check.